config/composer.json

```diff
@@ -12,24 +12,32 @@
         },
         {
             "type": "vcs",
-            "url": "git@github.com:pookmish/behat-tools.git"
+            "url": "https://github.com/pookmish/behat-tools.git"
         }
     ],
     "require": {
+        "php": ">=7.1",
+        "behat/mink-selenium2-driver": "^1.3.1",
         "bex/behat-screenshot": "^1.2",
+        "cweagans/composer-patches": "^1.6",
         "dmore/behat-chrome-extension": "^1.0.0",
         "drupal/core": "^8.8.0",
         "drupal/drupal-extension": "^4.0",
+        "drush/drush": "^10.2",
```

Comment:
Do we need drush 10? This might conflict with our stack repos as they need to be on Drush 9 at the moment.

Reply:
so this composer.json is used on the test builds when building a drupal directory from scratch. its not a dependency of this repo.